### PR TITLE
Release Notes for 7.7 (continued)

### PR DESCRIPTION
**Pull request by karenzone**
This PR is based off of the work in #11721 and incorporates additional review comments. 

**PREVIEW:** http://logstash_11762.docs-preview.app.elstc.co/guide/en/logstash/7.7/logstash-7-7-0.html

### CODE REVIEW

**Comment by karenzone** [comment]
@robbavey @andsel @roaksoax  Here are the release notes as they stand now.  We'll need to do another pass right before we release to pick up any plugin changes since Feature Freeze. We can do official reviews at that time.

**Comment by karenzone** [comment]
@robbavey @andsel @roaksoax @acchen97  

**UPDATE:**
No new plugin changes. Please review. Thanks!

File: docs/static/releasenotes.asciidoc [hunk 1]
```diff
@@ -27,6 +28,152 @@ This section summarizes the changes in the following releases:
...
+then forward it (internally) to the dedicated Monitoring cluster. Newer versions
+improved the process by allowing the data to be sent directly to the Monitoring
+cluster instead, in a different data format. Following those changes, Logstash
+has now added support to send the monitoring data directly to the Monitoring
+cluster, with the caveat that the Production cluster only has to retrieve the
+`cluster_uuid`.
+https://github.com/elastic/logstash/pull/11106[#11106],  
+https://github.com/elastic/logstash/pull/11640[#11640],
+https://github.com/elastic/logstash/pull/11641[#11641]  
+
```
Comment (andsel) [inline docs/static/releasenotes.asciidoc:55]: About `cluster_uuid` Rob also fixed a problem with hardcoded pipeline name issue: #11775 PR: #11777

Comment (roaksoax) [reply docs/static/releasenotes.asciidoc:55]: I think its ok not to include since that Bugfix was just an issue that appeared due to the changes above.

**Pull request merged**